bitfinex-client/src/client/bitfinex_response.rs

```diff
@@ -399,10 +459,17 @@ mod tests {
     #[test]
     fn wallet_details() {
         // let response_text = "[[\"exchange\",\"TESTBTC\",0.01,0,0.01,null,null],[\"exchange\",\"TESTUSD\",100,0,100,null,null],[\"exchange\",\"TESTUSDT\",200,0,200,null,null]]";
-        let response_text = "[[\"exchange\",\"TESTBTC\",0.005,0,0.005,null,null],[\"exchange\",\"TESTUSD\",100,0,100,null,null],[\"exchange\",\"TESTUSDT\",200,0,200,null,null],[\"margin\",\"TESTBTC\",0.005,0,0.005,null,null]]";
+        let response_text = "[[\"exchange\",\"TESTBTC\",1e-8,0,1e-8,null,null],[\"margin\",\"TESTUSD\",10921.63184294,0,0,\"Position #169742974 funding cost\",{\"reason\":\"POS_FUNDING_SETTLE\",\"pos_id\":169742974,\"pos_pair\":\"TESTBTC:TESTUSD\"}],[\"exchange\",\"TESTUSDT\",11000,0,11000,null,null],[\"exchange\",\"TESTUSD\",49999,0,49999,null,null]]";
```

Comment:
had to change the test vector because trading_details which is an optional JSON was not being handled properly.